### ISSUE TITLE
Calling any function generates false positive

### ISSUE THREAD

**Issue by aminya** [tonyhffong/Lint.jl#266]
If we want to use any function that is exported by other packages, we get this error.
For example:
```julia
using LanguageServer
LanguageServerInstance(stdin, stdout, true, named_pipe, "", Dict())
```
Gives:
```
: MethodError(convert, (Symbol, :($(Expr(:., :Pkg)))), 0x00000000000068b1)
```

As a quick fix please disable this error
